src/js/store/plans/actions.ts

```diff
@@ -97,6 +102,27 @@ export const startPreflight = (
     });
 };
 
+// export const createOrg = (planId, email) => console.log(planId, email);
+export const createOrg = (
```

Comment:
@jgerigmeyer I'm assuming that we can follow similar patterns as Metecho when spinning up scratch orgs.... I've hit a block here dispatching this `createOrg` button, doesn't seem to want to call this when adding the callback and curious if you can see what I'm missing here.  

Also, what are your thoughts on where these Org `actions`/ `reducers` should live? Whether they should be their own thing, or OK keeping here since attached to plans?

Reply:
@duggiemitchell 

> @jgerigmeyer I'm assuming that we can follow similar patterns as Metecho when spinning up scratch orgs.... I've hit a block here dispatching this `createOrg` button, doesn't seem to want to call this when adding the callback and curious if you can see what I'm missing here.

I pushed a fix for this. Because this project isn't using the `useDispatch` hook, Redux actions are attached via the `connect()` fn at the bottom of `plans/detail.tsx`. You had that correct, except actions need to be attached under a different name (e.g. `doCreateOrg`) to avoid namespace conflicts. So basically you were trying to call the imported action directly, instead of the _attached_ action as a prop.

> Also, what are your thoughts on where these Org `actions`/ `reducers` should live? Whether they should be their own thing, or OK keeping here since attached to plans?

Hm, that's a good question. It depends a bit on where these `orgs` will be stored in the Redux store eventually. My sense is that an org will be attached to a `job`. But I think I'd lean toward adding a new `scratch_org` top-level store module?